packages/graphql/tests/performance/server/servers/apollo.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+"use strict";
+
+// eslint-disable-next-line import/named
+import { ApolloServer } from "@apollo/server";
+import { startStandaloneServer } from "@apollo/server/standalone";
+import neo4j from "neo4j-driver";
+// eslint-disable-next-line import/no-unresolved
+import { Neo4jGraphQL } from "@neo4j/graphql";
+import { getLargeSchema } from "../typedefs.js";
+
+async function main() {
+    const driver = neo4j.driver("bolt://localhost:7687", neo4j.auth.basic("neo4j", "password"), {
```

Comment:
mmm, right now this is not really integrated into the ecosystem, because this runs with plain javascript + wrappers I haven't really spent time looking at how to integrate the env variables properly tbh

Reply:
```suggestion
    const { NEO_USER = "neo4j", NEO_PASSWORD = "password", NEO_URL = "bolt://localhost:7687" } = process.env;
    const driver = neo4j.driver(NEO_URL, neo4j.auth.basic(NEO_USER, NEO_PASSWORD), {
```

Fairly sure this would work without much of an issue.

Reply:
Oh, yeah,this works and I was planning on adding this. But you'll have to setup the env var variable manually when running the tests as the .env is not injected, that's what I meant, with integrated with the ecosystem.